src/urllib3/poolmanager.py

```diff
@@ -68,7 +83,9 @@
 PoolKey = collections.namedtuple("PoolKey", _key_fields)
```

Comment:
`PoolKey` will have to change into a `typing.NamedTuple`, I don't think that'll be too tough looking through `_key_fields`?